pkg/controllers/routing/network_routes_controller.go

```diff
@@ -718,8 +718,11 @@ func (nrc *NetworkRoutingController) startBgpServer() error {
 
 	nrc.bgpServer = gobgp.NewBgpServer()
 	go nrc.bgpServer.Serve()
-
-	g := bgpapi.NewGrpcServer(nrc.bgpServer, nrc.nodeIP.String()+":50051"+","+"127.0.0.1:50051")
+	ip := nrc.nodeIP.String()
+	if len(strings.Split(ip, ":")) >= 0 {
```

Comment:
@supu2 This logic will result in the IP being bracketed regardless of whether it's an IPv4 or IPv6 address as even if the string doesn't contain a colon `strings.Split` will result in a length of at least 1. See an example here: https://play.golang.org/p/k10huoTr9rE

Once this is updated to look for a `len` greater than 1 this seems fine to me. While I doubt that this is the only thing required for IPv6 to work in kube-router, it would fix a more glaring bug.

@murali-reddy I traced the code back, and it looks like GoBGP passes this string all the way back to net.dial.Listen in which bracketing an IPv6 address string is the correct thing to do.